main.nf

```diff
@@ -0,0 +1,32 @@
+// Ensure DSL2
+nextflow.enable.dsl = 2
+
+//test params
+params.config = "/agora-data-tools/test_config.yaml"
```

Comment:
Let's start moving test parameter values to configuration profiles like nf-core. Update your `nextflow.config` file as follows. You can then use the test values by running `nextflow run main.nf -profile test,docker`. 

```nextflow
profiles {

    docker {
        docker.enabled         = true
        docker.userEmulation   = true
        singularity.enabled    = false
        podman.enabled         = false
        shifter.enabled        = false
        charliecloud.enabled   = false
    }

    test {
        params {
            config = "/agora-data-tools/test_config.yaml"
        }
    }

}
```